demo_rails/app/assets/javascripts/application.js

```diff
@@ -13,3 +13,11 @@
 //= require rails-ujs
 //= require turbolinks
 //= require_tree .
+
+//= require highcharts/highcharts
+
+//= require googlecharts/google_visualr
+//= require googlecharts/loader
+
+//= require jquery-latest.min
+//= require jquery.dataTables
```

Comment:
new line ??

Reply:
Can you also comment out about these lines, why it is used. So that people can understand the code easily.